test/builtin/atomic/test_numbers.py

```diff
@@ -183,38 +188,40 @@ def test_n():
     [
         # Accuracy for 0
         ("0", "Infinity"),
-        ("0.", "323.607"),
-        ("0.00", "323.607"),
-        ("0.00`", "323.607"),
-        ("0.00`2", "323.607"),
-        ("0.00`20", "323.607"),
+        ("0.", ZERO_MACHINE_ACCURACY_STR),
+        ("0.00", ZERO_MACHINE_ACCURACY_STR),
+        ("0.00`", ZERO_MACHINE_ACCURACY_STR),
+        ("0.00`2", ZERO_MACHINE_ACCURACY_STR),
+        ("0.00`20", ZERO_MACHINE_ACCURACY_STR),
         ("0.00000000000000000000", "20."),
         ("0.``2", "2."),
         ("0.``20", "20."),
-        ("-0.`2", "323.607"),
-        ("-0.`20", "323.607"),
+        ("-0.`2", ZERO_MACHINE_ACCURACY_STR),
+        ("-0.`20", ZERO_MACHINE_ACCURACY_STR),
         ("-0.``2", "2."),
         ("-0.``20", "20."),
         # Now for non-zero numbers
         ("10", "Infinity"),
-        ("10.", "14.9546"),
-        ("10.00", "14.9546"),
-        ("10.00`", "14.9546"),
+        ("10.", DEFAUT_ACCURACY_10_STR),
+        ("10.00", DEFAUT_ACCURACY_10_STR),
+        ("10.00`", DEFAUT_ACCURACY_10_STR),
         ("10.00`2", "1."),
         ("10.00`20", "19."),
         ("10.00000000000000000000", "20."),
         ("10.``2", "2."),
         ("10.``20", "20."),
-        # Returns the accuracy of ```2.4```
-        (" 0.4 + 2.4 I", "15.5744"),
+        # For some reason, the following test
+        # would fail in WMA
+        ("1. I", "Accuracy[1.]"),
```

Comment:
Again this and the below are more understandable. Thanks. (And again this is shows how pytests are very different from tests that happen to get run in documentation.)